cmd/nfsmount/nfsmount.go

```diff
@@ -0,0 +1,50 @@
+// NFS mount is only needed for macOS since it has no
+// support for FUSE-based file systems
+//go:build darwin && !cmount
+// +build darwin !cmount
+
+package nfsmount
+
+import (
+	"context"
+	"fmt"
+	"net"
+	"os/exec"
+	"runtime"
+
+	"github.com/rclone/rclone/cmd/mountlib"
+	"github.com/rclone/rclone/cmd/serve/nfs"
+	"github.com/rclone/rclone/vfs"
+)
+
+func init() {
+	mountlib.NewMountCommand("mount", false, mount)
```

Comment:
We should make this available as `nfsmount` when it isn't being aliased to mount. Check out `cmd/cmount` for how it does it.